src/rhsm/connection.py

```diff
@@ -1257,15 +1291,15 @@ def hypervisorCheckIn(self, owner, env, host_guest_mapping, options=None):
 
             query_params = urlencode(params)
             url = "/hypervisors/%s?%s" % (owner, query_params)
-            res = self.conn.request_post(url, host_guest_mapping)
+            res = self.conn.request_post(url, host_guest_mapping, description=_("Registering hypervisor"))
             self.conn.headers["Content-type"] = priorContentType
         else:
             # fall back to original report api
             # this results in the same json as in the result_data field
             # of the new api method
             query_params = urlencode({"owner": owner, "env": env})
             url = "/hypervisors?%s" % (query_params)
-            res = self.conn.request_post(url, host_guest_mapping)
+            res = self.conn.request_post(url, host_guest_mapping, description=_("Registering hypervisor"))
```

Comment:
I think this might be more descriptive as "Updating detected virtual machines running on this host"

Reply:
Done.

Reply:
Same comment here.